recipe/recipe.yaml

```diff
@@ -1,30 +1,36 @@
-{% set name = "mdtraj" %}
-{% set version = "1.10.3" %}
+schema_version: 1
+
+context:
+  name: mdtraj
+  version: 1.10.3
 
 package:
   name: mdtraj
-  version: {{ version }}
+  version: ${{ version }}
```

Comment:
I got a good chuckle out of the release blog boasting how they got rid of Jinja templating

Reply:
Ha! Yes at least this is valid yaml, unlike stuff like this:
```
    {% if cuda_major >= 12 %}
    - cuda-version {{ cuda_compiler_version }}
    # for libcudadevrt.a
    - cuda-cudart-static
    # for cicc; split off starting from CUDA 12.2
    # - cuda-nvvm-tools
    - cuda-nvtx-dev
    - libcublas-dev
    - libcufft-dev
    - libcurand-dev
    - libcusolver-dev
    - libcusparse-dev
    {% endif %}
```

Reply:
I suppose so